tools/rdock/rdock.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="rdock" name="rDock" version="1.0">
+    <description>Binding Mode Prediction in Proteins/RNA</description>

```

Comment:
@bgruening I'm working on a `rdock` Bioconda recipe, but it depends on [cppunit](https://www.freedesktop.org/wiki/Software/cppunit/), which is not on any of Galaxy default channels. Seems like a candidate for conda-forge?